dist/dev/css/main-index.css

```diff
@@ -17,52 +17,35 @@ body {
         padding-right: 7px;
         padding-left: 7px;
     }
-
-    .col-lg-1 {
-        width: 76px;
-    }
-
-    .col-lg-2 {
-        width: 166px;
+    .container {
+        width: 1066px;
     }
 
-    .col-lg-3 {
-        width: 256px;
-    }
+}
 
-    .col-lg-4 {
-        width: 346px;
+@media (min-width: 992px) and (max-width: 1200px){
```

Comment:
From your previous logic, shouldn't max-width here be 1199px?

Reply:
i changed it because i thought u wanted to remove the hole between 1159 and  1200

Reply:
If media queries override, there isn't a point to setting max-width. Just stack the media queries in a certain order such that it'll override correctly. 
```
/* Extra Small Devices, Phones */ 
@media only screen and (min-width : 480px) {

}

/* Small Devices, Tablets */
@media only screen and (min-width : 768px) {

}

/* Medium Devices, Desktops */
@media only screen and (min-width : 992px) {

}

/* Large Devices, Wide Screens */
@media only screen and (min-width : 1200px) {

}
```

Reply:
do i need the "only screen" part as well?

Reply:
Just put it. We can get some nerd cred

Reply:
wait, i thought that the small screens had a max width of 768? or are we changing it to a min width of 480?

Reply:
leave it at max-width